### ISSUE TITLE
Suggest changing link to https

### ISSUE THREAD

**Issue by chmac** [alexgorbatchev/gpg-online#3]
I realise it doesn't really matter in the sense that the whole thing is done client side, but loading via HTTPS seems like a good idea, and is already supported.

I would submit a PR but I meant the URL listed on the homepage of the repository which can only be changed by the repo owner @alexgorbatchev.

PS> Thanks for this, great to have something verifiably open source for this purpose.

**Comment by RyanBalfanz** [comment]
Yep, https://alexgorbatchev.github.io/gpg-online/ works just fine, as @chmac noted.
